src/coreclr/interpreter/eeinterp.cpp

```diff
@@ -94,7 +94,7 @@ CorJitResult CILInterp::compileMethod(ICorJitInfo*         compHnd,
         if (InterpConfig.Interpreter().contains(compHnd, methodInfo->ftn, compHnd->getMethodClass(methodInfo->ftn), &methodInfo->args))
         {
             doInterpret = true;
-            g_interpModule = methodInfo->scope;
+            // g_interpModule = methodInfo->scope;
```

Comment:
@janvorli do we want to check this in? You mentioned needing to do it.

Reply:
No, this is just for testing the startup path. It is needed for the Interpreter test to work. I would introduce one more InterpMode that would behave like the mode 0, but it would prevent setting the g_interpModule. Or vice versa.